src/main/java/az/subscriptionMS/controller/SubscriptionController.java

```diff
@@ -0,0 +1,48 @@
+package az.subscriptionMS.controller;
+
+import az.subscriptionMS.model.Subscription;
+import az.subscriptionMS.service.SubscriptionsService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/v1/subscriptions")
+@RequiredArgsConstructor
+public class SubscriptionController {
+
+	@Autowired
```

Comment:
field based injection is not recommended change to constructor based

Reply:
OK